modules/internal/ChapelLocale.chpl

```diff
@@ -593,13 +737,14 @@ module ChapelLocale {
                   __primitive("array_get", origRL, 0), 
                   numLocales:size_t);
       // Set the rootLocale to the local copy
-      rootLocale = newRootLocale;
+      /*delete rootLocale._instance;*/
```

Comment:
Commented-out code always leads to the question "Why?"  Should this be uncommented, or maybe removed altogether?  If it should stay, and remain commented, we should add a description of why it's here but commented out.